From: Felipe Sateler <fsateler@gmail.com>
Date: Thu, 15 Apr 2010 14:47:26 -0400
Subject: [PATCH] Document changes applied in the debian package

Some documentation refers to the location of gettext files and
checkinstallrc, which we modify.
---
 NLS_SUPPORT |    4 ++--
 README      |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/NLS_SUPPORT b/NLS_SUPPORT
index cedebfb..231cc81 100644
--- a/NLS_SUPPORT
+++ b/NLS_SUPPORT
@@ -39,11 +39,11 @@ checkinstall-es.po
 3) Now create the locale directory for your language. Remember to use the same
    ISO codes you use in your LC_ALL environment variable:
 
-mkdir -p /usr/local/lib/checkinstall/locale/es_MX/LC_MESSAGES
+mkdir -p /usr/share/locale/es_MX/LC_MESSAGES
 
 4) Next, you should generate the .mo binary message file. Run msgfmt:
 
-msgfmt -o /usr/local/lib/checkinstall/locale/es_MX/LC_MESSAGES/checkinstall.mo \
+msgfmt -o /usr/share/locale/es_MX/LC_MESSAGES/checkinstall.mo \
      checkinstall-es.po
 
 (The above goes all in one line)
diff --git a/README b/README
index 086aee7..6f4c605 100644
--- a/README
+++ b/README
@@ -343,7 +343,7 @@ CHANGE in CheckInstall 1.4.0:
 
  The variable declaration section in the checkinstall script has been removed,
  you should now edit the checkinstallrc file, normally installed under 
- /usr/local/lib/checkinstall.
+ /usr/local/lib/checkinstall (/etc in this Debian package).
 
 
  In the checkinstallrc file you will find some variables wich modify
-- 
